app/uk/gov/hmrc/cardpaymentfrontend/views/PaymentFailedPage.scala.html

```diff
@@ -78,7 +79,7 @@ <h1 class="govuk-heading-xl">@{
     <ul class="govuk-list--bullet govuk-!-static-padding-left-7">
         <li class="govuk-body" id="line2">@{messages("payment-failed.failed2")}</li>
         <li class="govuk-body" id="line3">@{messages("payment-failed.failed3")}</li>
-        @if(taxType != "pngr"){ <li class="govuk-body" id="line4">@{messages("payment-failed.failed4")}</li> }
+        @if(origin.toTaxType.toString != "pngr"){ <li class="govuk-body" id="line4">@{messages("payment-failed.failed4")}</li> }
```

Comment:
why not origin != Origins.pngr or whateve it is, you're converting it to a string to compare it to a string